src/pkg/repository/repository_unexported_test.go

```diff
@@ -51,3 +109,186 @@ func (suite *RepositoryModelSuite) TestWriteGetModel() {
 	require.NoError(t, err)
 	assert.Equal(t, "fnords", string(got.ID))
 }
+
+// helper func for writing backups
+//
+//revive:disable:context-as-argument
+func writeBackup(
+	t *testing.T,
+	ctx context.Context,
+	kw *kopia.Wrapper,
+	sw *store.Wrapper,
+	tID, snapID, backupID string,
+	sel selectors.Selector,
+	deets *details.Details,
+	errors fault.Errors,
+	errs *fault.Bus,
+) *backup.Backup {
+	//revive:enable:context-as-argument
+	var (
+		serv         = sel.PathService()
+		detailsStore = streamstore.NewDetails(kw, tID, serv)
+		errorsStore  = streamstore.NewFaultErrors(kw, tID, serv)
+	)
+
+	detailsID, err := detailsStore.Write(ctx, deets, errs)
+	require.NoError(t, err, "writing details to streamstore")
+
+	errorsID, err := errorsStore.Write(ctx, errs.Errors(), errs)
```

Comment:
shouldn't use the same input and error collector here either

Reply:
tis safe to do.